pennylane_lightning/src/Bindings.cpp

```diff
@@ -18,45 +18,65 @@
 #include "pybind11/pybind11.h"
 #include "pybind11/stl.h"
 
+/// @cond DEV
```

Comment:
What does this bit do? :thinking:

Reply:
This is understood as a conditional option for Doxygen --- anything within the `@cond LABELNAME` regions will be excluded for generating API docs. We should be publishing the binding API with the C++ API, so this just ensures that entire region is ignored.